text/message_system.go

```diff
@@ -13,3 +13,11 @@ func NewErrorSystemGeneric(reason string) *Message {
 		}),
 	}
 }
+
+func NewErrorSelfServiceNoMethodsAvailable() *Message {
```

Comment:
I don't think that we should add any element specific messages here. Kratos is headless, and should only care about the messages it uses.

In the previous Ory elements version, we also had two sources of messages for translations, and had a small script to merge the two:
https://github.com/ory/elements/blob/a93569f94b252ba193658199559f944162f9b6ab/scripts/generate-locales.ts#L53-L80

I strongly suggest to not add any messages here that are not part of this codebase, as it will become a nightmare to maintain down the road.